spellmonitor.lic

```diff
@@ -1,15 +1,12 @@
-# quiet
 =begin
   Documentation: https://elanthipedia.play.net/Lich_script_development#spellmonitor
 =end
 
-custom_require.call %w[drinfomon]
+custom_require.call(%w[drinfomon])
 no_kill_all
 no_pause_all
-# hide_me
-setpriority(0)
 
-$CYCLICAL = 1001
+setpriority(0)
```

Comment:
I don't know the purpose why this script sets the priority to 0 -- is this to make the "low level" scripts run _before_ the other lich scripts?